cmd/login/cmd.go

```diff
@@ -240,8 +244,12 @@ func run(cmd *cobra.Command, argv []string) {
 		Logger(logger).
 		Build()
 	if err != nil {
-		reporter.Errorf("Failed to create OCM connection: %v", err)
-		os.Exit(1)
+		if strings.Contains(err.Error(), "token needs to be updated") {
+			reattemptLogin(cmd, argv)
```

Comment:
Before calling this, should we be checking `if force == true` and just output the error instead? Want to make sure we don't end up in an infinite loop reattempting login